lib/jsonapi/serializer/version.rb

```diff
@@ -0,0 +1,5 @@
+module JSONAPI
+  module Serializer
+    VERSION = '2.0.0'.freeze
+  end
```

Comment:
I like the idea of nesting this in Serializer, it allows for sharing the JSONAPI module/namespace.